libs/react-components/tedi/src/components/closing-button/closing-button.tsx

```diff
@@ -0,0 +1,60 @@
+import cn from 'classnames';
```

Comment:
I think we should also use this in Modal, currently there is custom Button

Reply:
Agreed, I looked into Figma, Modal component was actually ready for dev so It would make sense to replace it during the Modal development, I would rather not break anything in Community's modal in this PR :D